libs/wingc/src/jsify.rs

```diff
@@ -324,7 +324,12 @@ impl JSifier {
 					return self.jsify_global_utility_function(&args, UtilityFunctions::Throw);
 				} else if let Reference::NestedIdentifier { object, .. } = function {
 					let object_type = object.evaluated_type.borrow().unwrap();
-					needs_case_conversion = object_type.as_class_or_resource().unwrap().should_case_convert_jsii;
+					needs_case_conversion = if let Some(obj) = object_type.as_class_or_resource() {
+						obj.should_case_convert_jsii
+					} else {
+						// This should only happen in the case of `any`, which are almost certainly JSII imports.
```

Comment:
Just to be sure, even in case of JSII imports this shouldn't normally happen. It might be required because of unimplemented stuff. CMIIW.

Assuming I'm right, I had this thought that it might be good to print some notice level report in these cases, just so we're aware of what's happening. Makes sense?

Reply:
After checking again I opted to keep the same logic but I made the comment more accurate (I hope).